src/layout/_page.scss

```diff
@@ -14,3 +14,7 @@
     padding: ms(10) ms(2);
   }
 }
+
+.page-section--compact {
```

Comment:
Agree, chain or nest. We don't do this everywhere now. @misscs, Can we get an opinion?